CMakeLists.txt

```
@@ -1,40 +1,43 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.11)
 project(ostis-metasystem)
 set(CMAKE_CXX_STANDARD 17)
 
-option(SC_BUILD_TESTS "Flag to build unit tests" OFF)
+option(SC_BUILD_TESTS "Flag to build unit tests" ON)
+option(SC_CLANG_FORMAT_CODE "Flag to add clangformat and clangformat_check targets" OFF)
 
 if(NOT SC_BIN_PATH)
     set(SC_BIN_PATH "${CMAKE_BINARY_DIR}/../bin")
 endif()
 set(SC_EXTENSIONS_DIRECTORY "${SC_BIN_PATH}/extensions")
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${SC_BIN_PATH})
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${SC_EXTENSIONS_DIRECTORY})
 link_directories(${SC_BIN_PATH} ${SC_EXTENSIONS_DIRECTORY})
 
 set(SC_MACHINE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/ostis-web-platform/sc-machine")
-set(SC_CODEGEN_TOOL "${SC_BIN_PATH}/sc-code-generator")
 set(CMAKE_MODULE_PATH "${SC_MACHINE_PATH}/cmake")
```

I would've used append for idempotent behavior on include order changes
```suggestion
list(APPEND CMAKE_MODULE_PATH "${SC_MACHINE_PATH}/cmake")
```